demo/admin/src/documents/pages/blocks/SliderBlock.tsx

```diff
@@ -0,0 +1,40 @@
+import { BlockCategory, createCompositeBlock, createListBlock } from "@comet/blocks-admin";
+import { MediaBlock } from "@src/common/blocks/MediaBlock";
+import { RichTextBlock } from "@src/common/blocks/RichTextBlock";
+import { FormattedMessage } from "react-intl";
+
+const SliderItemBlock = createCompositeBlock({
+    name: "SliderItem",
+    displayName: <FormattedMessage id="sliderItemBlock.displayName" defaultMessage="Slider Item" />,
+    category: BlockCategory.Teaser,
+    blocks: {
+        media: {
+            block: MediaBlock,
+            title: <FormattedMessage id="sliderItemBlock.media" defaultMessage="Media" />,
+            hiddenInSubroute: true,
+        },
+        text: {
+            block: RichTextBlock,
+            hiddenInSubroute: true,
+        },
+    },
+});
+
+export const SliderListBlock = createListBlock({
```

Comment:
We don't need to export this:
```suggestion
const SliderListBlock = createListBlock({
```